src/shared/cloudformation/cloudformation.ts

```diff
@@ -384,6 +411,47 @@ export namespace CloudFormation {
         return template
     }
 
+    export async function tryLoad(
```

Comment:
Something like this to clarify the conditions

```suggestion
   /** 
   * Returns the parsed the content of the given uri, but if the URI does not exist on disk (untitled scheme) it 
   * parses the given content */
    export async function tryLoad(
```